src/lib/y2storage/filesystems/base.rb

```diff
@@ -45,6 +45,10 @@ class Base < Mountable
       #   @return [Filesystems::Type]
       storage_forward :type, as: "Filesystems::Type"
 
+      # @!method supports_mount?
+      #   @return [Boolean]
+      storage_forward :supports_mount?
```

Comment:
I expected this forward method directly in `Mountable` class.

Reply:
Indeed.